sdk/python/setup.py

```diff
@@ -30,7 +30,7 @@
     # kfp-server-api package is released. Update the lower version when there is
     # a breaking change in kfp-server-api, or kfp sdk depends on new api changes
     # in kfp-server-api.
-    'kfp-server-api>=0.2.5, <0.4.0',
+    'kfp-server-api>=0.2.5, <0.6.0',
```

Comment:
good catch, thanks

Reply:
`requirements.in` and `requirements.txt` need to be updated as well.